src/components/SlideDeck/SlideDeck.js

```diff
@@ -14,6 +14,7 @@ import { Fullscreen, KeyboardArrowLeft, KeyboardArrowRight } from "@material-ui/
 
 class Slide extends React.Component {
 	static propTypes = {
+        customRef: PropTypes.string,
```

Comment:
```suggestion
        name: PropTypes.string,
```
"ref" leaves room for confusion or misinterpretation.